solidity/contracts/deposit/Deposit.sol

```diff
@@ -192,15 +192,15 @@ contract Deposit is DepositFactoryAuthority {
     /// @dev                Will revert if redemption is not possible by _redeemer.
     /// @param _redeemer    The deposit redeemer.
     /// @return             The amount in TBTC needed to redeem the deposit.
-    function getRedemptionTbtcRequirement(address _redeemer) public view returns(uint256){
+    function getRedemptionTbtcRequirement(address _redeemer) public view returns (uint256){
```

Comment:
`calculateRedemptionTbtcAmounts` is deliberately designed as an internal function, and its results are exposed publicly in a very controlled way. TDT and FRT payouts are implementation details for redeemers, so I'm not convinced we need to publicly expose anything beyond what's already public here. I could see us removing the `DepositUtils` version of the two functions in favor of calling `calculateRedemptionTbtcAmounts` directly from `Deposit`'s implementation though.

Reply:
> TDT and FRT payouts are implementation details for redeemers

We don't necessarily need to expose these, the public declaration could just return `owedToDeposit`. 
> calculateRedemptionTbtcAmounts is deliberately designed as an internal function, and its results are exposed publicly in a very controlled way

It would still be controlled, but it would be one function instead of two. The only thing we would lose is the naming distinction, but it would reduce the total number of functions.

Either way it's kind of a nit

Reply:
I think we might be on the same page. 

> We don't necessarily need to expose these, the public declaration could just return owedToDeposit.

This is already what happens.

I'm suggesting we change the public functions on `Deposit` to directly call `calculateRedemptionTbtcAmounts`, and drop the duplicate `getRedemptionTbtcRequirement`/`getOwnerRedemptionTbtcRequirement` on `DepositRedemption` as redundant. Is that what you're suggesting as well?

Reply:
that's sounds good, if we want to keep the naming distinction.

Reply:
Kk, pushed this tweak.